sys/cheri/cheri_sysctl.c

```diff
@@ -37,7 +37,7 @@
 #include <cheri/cheri.h>
 
 #ifdef __CHERI_PURE_CAPABILITY__
-FEATURE(cheriabi_kernel, "CheriABI kernel");
+FEATURE(cheri_purecap_kernel, "pure-capability kernel");
 #ifdef __CHERI_SUBOBJECT_BOUNDS__
 FEATURE(subobject_bounds, "CheriABI kernel with sub-object bounds");
```

Comment:
```suggestion
FEATURE(subobject_bounds, "CHERI pure-capability kernel with sub-object bounds");
```

I think Konrad's suggestion of renaming this to include "cheri" might also be useful.